src/main/java/org/dbtools/query/shared/QueryCompareType.java

```diff
@@ -10,5 +10,5 @@
 package org.dbtools.query.shared;
 
 public enum QueryCompareType {
-    EQUAL, NOT_EQUAL, LESSTHAN, GREATERTHAN, LESSTHAN_EQUAL, GREATERTHAN_EQUAL, LIKE, LIKE_IGNORECASE, IN, IS_NULL, NOT_NULL
+    EQUAL, NOT_EQUAL, LESSTHAN, GREATERTHAN, LESSTHAN_EQUAL, GREATERTHAN_EQUAL, LIKE, LIKE_IGNORECASE, IN, IS_NULL, NOT_NULL, NONE
```

Comment:
Should probably make "NONE" the first enum (would look weird to add more enum values later and have "NONE" in the middle of a lot of other enums)